unified-installer/astra-unified-installer.sh

```diff
@@ -2133,136 +2340,14 @@ step_init_generated_dirs_and_files() {
 
     rm -rf "$kustomization_dir" "$crs_dir"
     mkdir -p "$crs_dir"
-    mkdir -p "$kustomization_dir"
 
-    if [ ! -f "$kustomization_file" ]; then
-        cat <<EOF > "$kustomization_file"
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-resources:
-patches:
-transformers:
-EOF
-        logdebug "$kustomization_file: OK"
-    fi
-}
-
-step_kustomize_global_namespace_if_needed() {
```

Comment:
This step got turned into a generic function which is called from the `step_generate_whatever_yaml` function for the connector and trident.